index.html

```diff
@@ -55,9 +56,17 @@ <h2 class="entry-header">New Entry</h2>
               <button class="button" type="submit">SAVE</button>
             </div>
           </div>
-          <div data-view="entries" class="hidden"></div>
         </form>
       </div>
+      <div id="entries" data-view="entries" class="hidden">
+        <div class="row entries-header">
+          <h1>Entries</h1>
+          <button class="new-entry-button">NEW</button>
+        </div>
+        <ul class="past-entries">
+          <li class="no-entries">No entries have been recorded</li>
```

Comment:
This should not be inside of the `ul`. The `ul` is reserved for the entries that will be generated